app/controllers/observation_quality_experiments_controller.rb

```diff
@@ -0,0 +1,101 @@
+# frozen_string_literal: true
+
+class ObservationAccuracyExperimentsController < ApplicationController
+  layout "bootstrap"
+
+  def get_more_validators
+    experiment_id = params[:id]
+    @experiment = ObservationAccuracyExperiment.find( experiment_id )
```

Comment:
If an invalid ID is provided, this will raise an error. We handle this in different ways in different controllers. See for example the Announcements controller https://github.com/inaturalist/inaturalist/blob/main/app/controllers/announcements_controller.rb#L6 where we use a before_action to attempt to load the record, or raise a 404 if the record is not found